### DISCUSSION TITLE
Include list of downloaded songs in backup

### DISCUSSION THREAD

**Discussion by kaneryu** [z-huang/music#1245] [General]
Basically, I want this [table](https://github.com/z-huang/InnerTune/blob/6660e8b774303d2464f06b7c09f0768ecb19c014/app/src/main/java/com/zionhuang/music/db/MusicDatabase.kt#L206) to be included in backups. The reason is that I use the downloaded songs as my main way of accessing my music, without using any playlists. The issue is that when I do an export, I can't acess the list of downloaded songs in the backup.

**Comment by wl0t** [comment]
Hi, do you know how to read the content of the .backup file ? It doesn't seem to be a sql database file directly.

**Reply by kaneryu** [reply]
Yes, It's a ZIP file containing a settings.json and the SQLite database

**Reply by wl0t** [reply]
thank you !
